storybook/stories/api/offset.js

```diff
@@ -1,39 +0,0 @@
-// @flow
```

Comment:
Yeah I think there's value to have it, particularly given that most 12 column grid systems offer it